src/bsk201903/connectMultipleClusters.md

```diff
@@ -215,14 +242,14 @@ If you're interested in diving more into the topic, you should have a look at th
 - The retailer Target is using [Spinnaker to orchestrate deployment across several clusters](https://tech.target.com/infrastructure/2018/06/20/enter-unimatrix.html)
```

Comment:
unimatrix is more like a cluster aggregating facade for Spinnaker though, Target does not only use Spinnaker (that's the orchestration engine) - unimatrix is the actual component that aggregates clusters